phpstan.neon.dist

```diff
@@ -1,4 +1,5 @@
 includes:
+	- phar://%rootDir%/phpstan.phar/conf/bleedingEdge.neon
```

Comment:
Seems this does not work

Reply:
yes, bleeding edge rules are there from 0.11.9...
i will wait for https://github.com/nextras/orm/pull/355 and then rebase this pr

Reply:
Thanks, done.